cpp/ql/src/experimental/Best Practices/WrongUintAccess.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @id cpp/wrong-uint-access
+ * @name Wrong Uint
+ * @descripion Acess an array of size lower than 256 with a uint16.
+ * @kind problem
+ * @problem.severity recommendation
+ * @tags efficiency
+ */
+
+import cpp
+import semmle.code.cpp.controlflow.SSA
+
+from
+  Variable E, ArrayExpr useExpr, ArrayType defExpr, VariableDeclarationEntry def, VariableAccess use
```

Comment:
Two comments about this line:
- We're _just_ in the process of deprecating uppercase identifiers. See https://github.com/github/codeql/pull/8930. So I'd appreciate it if you'd rename `E` to something lowercase. `v` is always a great name for a `Variable` if you can't think of anything better 😂
- I find it slightly strange that `defExpr` is a _type_ (and not an `Expr` like `useExpr`).

Reply:
Done
But I'm not sure for the name of defExpr that I changed to defLine. I think that it's more understandable but I'm not sure if it's enough.

Reply:
I would probably have gone with `arrayType` instead of `defLine`. But at this point, we're just splitting hairs :) So this is all fine!